src/metabase/notification/send.clj

```diff
@@ -62,12 +63,17 @@
                               (try
                                 (channel/send! channel message)
                                 (catch Exception e
-                                  (when (should-retry-sending? e (:type channel))
+                                  (let [skip-retry? (should-skip-retry? e (:type channel))
+                                        new-e       (ex-info (ex-message e)
+                                                             (assoc (ex-data e) ::skip-retry? skip-retry?))]
                                     (vswap! retry-errors conj {:message   (u/strip-error e)
```

Comment:
I think you meant to skip this if it's not retry-able

Reply:
eagle eyes!